File: integration-tests/TestAPI-bindings/c-sharp/TestAPI.Tests/ByteTests.cs
using System;
using Xunit;
using Cricut.TestAPI;

namespace TestAPI.Tests {
    public class ByteTests {
        [Fact]
        void testBytesValues() {
            Assert.Equal(Bytes.bytes, new byte[] { 0xDE, 0xAD, 0xBE, 0xEF });
            Assert.Equal(Bytes.data, new byte[] { 0x0B, 0xAD, 0xF0, 0x0D });
        }

        [Fact]
        void testEchoBytes() {
            Assert.Equal(Bytes.echoBytes(Bytes.bytes), Bytes.bytes);
            Assert.Equal(Bytes.echoData(Bytes.data), Bytes.data);
        }
    }
}

File: integration-tests/TestAPI/bindings/kotlin/src/test/kotlin/com/cricut/testapi/ByteTests.kt
package com.cricut.testapi

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test
import java.util.*

internal class ByteTests {
    @Test
    fun testBytesValues() {
        assertEquals(Bytes.bytes, arrayListOf(0xDE.toUByte(), 0xAD.toUByte(), 0xBE.toUByte(), 0xEF.toUByte()))
        assert(Bytes.data.contentEquals(byteArrayOf(0x0B.toByte(), 0xAD.toByte(), 0xF0.toByte(), 0x0D.toByte())))
    }

    @Test
    fun testEchoBytes() {
        assertEquals(Bytes.echoBytes(Bytes.bytes), Bytes.bytes)
        assert(Bytes.echoData(Bytes.data).contentEquals(Bytes.data))
    }
}